controllers/configurationpolicy_controller.go

```diff
@@ -983,13 +995,23 @@ func (r *ConfigurationPolicyReconciler) handleObjects(
 
 		log.V(2).Info("Handling a single object template")
 
-		objNames, compliant, rsrcKind, statusUpdateNeeded = r.handleSingleObj(
+		var creationInfo *policyv1.ObjectProperties
+
+		objNames, compliant, rsrcKind, statusUpdateNeeded, creationInfo = r.handleSingleObj(
```

Comment:
Please rebase on top of https://github.com/open-cluster-management-io/config-policy-controller/pull/47, I think there will be a conflict here.